capture-sdk/sdk/src/main/java/net/gini/android/capture/camera/CameraFragmentListener.java

```diff
@@ -31,6 +31,16 @@ interface CameraFragmentListener {
      */
     void onDocumentAvailable(@NonNull Document document);
 
+
+    /**
+     * <p>
+     * Called when starting MultiPageReview Screen
+     * </p>
+     *
+     * @param multiPageDocument the image taken by the camera or the validated imported document
+     * @param shouldScrollToLastPage if pages should scroll to the last one
```

Comment:
[Suggestion] Maybe it is worth mentioning that `shouldScrollToLastPage` is a flag that defines if pages should scroll to the last one?